OpenRA.Mods.Common/Activities/Move/MoveAdjacentTo.cs

```diff
@@ -66,6 +83,17 @@ public override Activity Tick(Actor self)
 		{
 			var targetIsValid = Target.IsValidFor(self);
 
+			// Target moved under the shroud. Move to its last known position.
```

Comment:
The comment says 'shroud', but checks ```HiddenUnderFog```. I think the comment should reflect that and say 'shroud or fog'.